execution/process.go

```diff
@@ -3,9 +3,11 @@ package execution
 import "os"
 
 type Process interface {
+	Container() *Container
```

Comment:
Yeah, hence why I thought I had removed it :). It's not used, I'll quickly delete it.